src/oas3/__tests__/service.test.ts

```diff
@@ -412,6 +429,7 @@ describe('oas3 service', () => {
           altText: 'logo',
           href: 'https://stoplight.io',
         },
+        infoExtensions: { 'x-logo': {} },
```

Comment:
@matthewmurphy , I think **nine** of the files changed in this PR are solely because we're doing snapshot or test-the-whole-result style testing.  I wonder when we should consider making some of these tests in `http-spec` more targeted.  🤔

Reply:
a snapshot test would have been great bc i could have had jest update them after justifying the change